webtask.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Auth0 Account Link",
   "name": "auth0-account-link",
-  "version": "2.6.4",
+  "version": "2.6.5",
```

Comment:
Please fix the preVersion as well.

Reply:
I'm thinking to keep `preVersion` as 2.6.3, because 2.6.4 was never put in prod:
![image](https://user-images.githubusercontent.com/71466100/124473017-cb2f1900-dd6c-11eb-9e55-6255d5111294.png)